pycircstat2/data/corner-case/C1.csv

```diff
@@ -0,0 +1,19 @@
+,alpha

```

Comment:
If the data is as simple as this, we don't need to add it as a dataset, but simply put, `alpha = np.arange(18)` within the `test_circ_median_ci()`. Moreover, I tried this but it doesn't break the original code. Maybe I didn't make it clear enough before: can you provide a case of artificial data which breaks the original code?